Snowflake.Data.Tests/SFPutGetTest.cs

```diff
@@ -2,418 +2,259 @@
+                    internalStage = $"@{schemaName}.%{TEST_TEMP_TABLE_NAME}{stagePath}";
+                    break;
+                case StageType.NAMED:
+                    internalStage = $"@{schemaName}.{TEST_TEMP_STAGE_NAME}{stagePath}";
+                    break;
+            }
+            
+            using (var conn = new SnowflakeDbConnection(ConnectionString))
             {
-                conn.ConnectionString = ConnectionString
-                    + String.Format(
-                    ";GCS_USE_DOWNSCOPED_CREDENTIAL=true");
                 conn.Open();
-
-                // Create a temp file
-                string filePath = Path.GetTempPath() + Guid.NewGuid().ToString() + ".csv";
-                // Write row data to temp file
-                File.WriteAllText(filePath, ROW_DATA);
-
-                string tempDirectory = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
-                Directory.CreateDirectory(tempDirectory);
-
-                string putQuery = $"PUT file://{filePath} @{DATABASE_NAME}.{SCHEMA_NAME}.%{TEST_TEMP_TABLE_NAME}";
-
-                string getQuery = $"GET @{DATABASE_NAME}.{SCHEMA_NAME}.%{TEST_TEMP_TABLE_NAME} file://{tempDirectory}";
-
-                string fileName = "";
-                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
-                {
-                    fileName = filePath.Substring(filePath.LastIndexOf('\\') + 1);
-                }
-                else
-                {
-                    fileName = filePath.Substring(filePath.LastIndexOf('/') + 1);
-                }
-
-                // Windows user contains a '~' in the path which causes an error
-                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
-                {
-                    putQuery = $"PUT file://C:\\\\Users\\{Environment.UserName}\\AppData\\Local\\Temp\\{fileName} @{DATABASE_NAME}.{SCHEMA_NAME}.%{TEST_TEMP_TABLE_NAME}";
-                }
-                putQuery += " AUTO_COMPRESS=FALSE";
-
-                using (DbCommand command = conn.CreateCommand())
+                using (var command = conn.CreateCommand())
                 {
-                    // Create temp table
-                    command.CommandText = createTable;
-                    command.ExecuteNonQuery();
-
-                    // Create temp stage
-                    command.CommandText = createStage;
-                    command.ExecuteNonQuery();
+                    // ********************************************************************
+                    // ** 1. PUT - upload file from local directory to the stage
+                    // ********************************************************************
+                    
+                    // Prepare PUT query - Windows user contains a '~' in the path which causes an error
+                    string putQuery = $"PUT file://{inputFilePath} {internalStage} AUTO_COMPRESS=" + (autoCompress ? "TRUE" : "FALSE");
```

Comment:
why `autoCompress ? "TRUE" : "FALSE"` is not part of interpolated string but concatenated separately?